src/main/java/duke/tasks/Event.java

```diff
@@ -0,0 +1,55 @@
+package duke.tasks;
+import java.time.*;
+import java.time.format.*;
+
+public class Event extends Task {
+    private LocalDateTime start;
+    private LocalDateTime end;
+
+    //Constructor
+    public Event (String name, LocalDateTime start, LocalDateTime end) {
```

Comment:
Should there be a space between the class name and the argument bracket?
```suggestion
    public Event(String name, LocalDateTime start, LocalDateTime end) {
```